javascript/javascript3/week2/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css">
+    <title>Currency calculator</title>
+</head>
+
+<body>
+    <main>
+        <header>
+            <h1>Currency calculator</h1>
+        </header>
+
+        <form action="#">
+            <div class="margin">
+                <label for="amount">Amount to convert: </label> <br>
+                <input type="number" id="amount" name="amount" value="1">
+
+            </div>
+
+            <div class="currency-container">
+                <div class="space">
+                    <label for="from-currency">From: </label> <br>
+                    <select name="from-currency" id="from-currency">
+                    </select>
+                </div>
+    
+                <div class="space">
+                    <label for="to-currency">To: </label> <br>
+                    <select name="to-currency" id="to-currency">
+                    </select>
+                </div>
+            </div>
+
+            <div id="convertedAmount">Converted Amount...</div>
+
+            <div class="button-container">
+                <button id="convert">Convert</button>
```

Comment:
It would be nice if reset and convert buttons look different. It would give users a better experience. You can read a bit more about that here https://uxplanet.org/primary-secondary-action-buttons-c16df9b36150